backend/scripts/setup_test_simulations.py

```
@@ -0,0 +1,192 @@
+
+# Add backend to path
+sys.path.append(str(Path(__file__).parent.parent))
+
+from database.connection import get_db_session
+from database.models import (
+    User, Cohort, Scenario, CohortStudent, CohortSimulation, 
+    StudentSimulationInstance, UserProgress
+)
+from utilities.id_generator import generate_unique_simulation_instance_id
+
+def setup_test_simulations(cohort_id: int = None, scenario_id: int = None, count: int = 40):
```

Ruff (0.14.4)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)